src/rethinkdb/utils.cljc

```diff
@@ -30,3 +30,7 @@
 
 (defn snake-case [s]
   (string/replace (name s) \- \_))
+
+(defn keyword->str [k]
+  (-> (str k)
```

Comment:
name removes the namespace from the keyword. This is exactly what I want to prevent.